Assets/Scripts/Editor/Compilation/DotnetExeCompilator.cs

```diff
@@ -27,6 +27,7 @@ public class DotnetExeDynamicCompilation: DynamicCompilationBase
 
         private static string ApplicationContentsPath = EditorApplication.applicationContentsPath;
         private static readonly List<string> _createdFilesToCleanUp = new List<string>();
+        private static readonly Dictionary<string, Assembly> _typeNameAssemblyCache = new(16);
```

Comment:
can we adjust to old init way. FSR is still supporting some older Unity versions like 2018, I think this new language feature will cause compilation error there

Reply:
Great point. I need to do a thorough review for C# version compatibility in all my PRs actually, as I habitually use modern features quite a lot.